src/shogun/kernel/SigmoidKernel.cpp

```diff
@@ -57,9 +57,8 @@ void CSigmoidKernel::init()
 	gamma = 0.0;
 	coef0 = 0.0;
 
-	SG_ADD(
+	sg_add<ParameterProperties::HYPER | ParameterProperties::AUTO>(
```

Comment:
what about moving the flags to the template deduction so that we can use a static_assert like in the macro?

Reply:
+

Reply:
is this a typo or is it not rendering in my browser?